### PR TITLE
[WIP] Refactor Omnichannel Past Chats list

### PR DESCRIPTION
**Pull request by nitinkumartiwari**
Closes #ISSUE_NUMBER

### CODE REVIEW

File: .meteor/packages [hunk 1]
```diff
@@ -97,3 +97,4 @@ webapp-hashing@1.0.9
 rocketchat:oauth2-server
 rocketchat:i18n
 dandv:caret-position
+twbs:bootstrap
```
Comment (renatobecker) [inline .meteor/packages]: Why do you need this?

File: .meteor/versions [hunk 2]
```diff
@@ -148,6 +148,7 @@ templating-tools@1.1.2
 tmeasday:check-npm-versions@0.3.2
 todda00:friendly-slugs@0.6.0
 tracker@1.2.0
+twbs:bootstrap@3.3.6
```
Comment (renatobecker) [inline .meteor/versions]: Why do you need this?

File: .meteor/packages (hunk 1 above)
Comment (nitinkumartiwari) [reply .meteor/packages]: I have used glyphicon in search option that is why i have added this package

File: .meteor/versions (hunk 2 above)
Comment (nitinkumartiwari) [reply .meteor/versions]: I have used glyphicon in search option it is updated package of bootstrap

File: .meteor/packages (hunk 1 above)
Comment (renatobecker) [reply .meteor/packages:100]: We have all the resources we need, natively.
Please, remove this dependency, thanks.

File: .meteor/versions (hunk 2 above)
Comment (renatobecker) [reply .meteor/versions:151]: We have all the resources we need, natively.
Please, remove this dependency, thanks.

File: app/ui/client/views/app/room.js [hunk 3]
```diff
@@ -318,7 +322,10 @@ Template.room.helpers({
 				ts: 1,
 			},
 		};
-
+		ChatMessage.update({rid:'12121212',msg:'tthis is updated message'}, options)
+		console.log('query',query);
+		console.log('optios',options)
+	//	console.log('=======',ChatMessage.collection._docs._map);
```
Comment (renatobecker) [inline app/ui/client/views/app/room.js:328]: Please, remove it.

File: app/ui/client/views/app/room.js [hunk 4]
```diff
@@ -1187,7 +1194,7 @@ Template.room.onRendered(function() {
 	}
 	chatMessages[rid].initializeWrapper(this.find('.wrapper'));
 	chatMessages[rid].initializeInput(this.find('.js-input-message'), { rid });
-
+console.log("chatmsg --",chatMessages);
```
Comment (renatobecker) [inline app/ui/client/views/app/room.js:1197]: Please, remove it.

File: app/utils/client/lib/RestApiClient.js [hunk 5]
```diff
@@ -9,7 +9,7 @@ export const APIClient = {
 		return APIClient._jqueryCall('DELETE', endpoint, params);
 	},
 
-	get(endpoint, params) {
+	get(endpoint, params) { console.log('endpoint2:'+endpoint+' para2 : '+params);
```
Comment (renatobecker) [inline app/utils/client/lib/RestApiClient.js:12]: Please, remove the console code.

File: app/livechat/client/views/app/tabbar/externalSearch.js [hunk 6]
```diff
@@ -45,6 +45,7 @@ Template.externalSearch.onCreated(function() {
 		this.roomId = Template.currentData().rid;
 		if (this.roomId) {
 			const { messages, total } = await APIClient.v1.get(`livechat/messages.external/${ this.roomId }?count=${ MESSAGES_COUNT }&offset=${ offset }`);
+			console.log("messagessss::::::::"+messages);
```
Comment (renatobecker) [inline app/livechat/client/views/app/tabbar/externalSearch.js:48]: Please, remove it.

File: app/livechat/client/views/app/tabbar/visitorHistory.js [hunk 7]
```diff
@@ -4,38 +4,84 @@ import moment from 'moment';
...
+let  historyResult 
+let len, msgs
+let allHistoryChat ;
 
-Template.visitorHistory.helpers({
+Template.allChatHistory.helpers({
+	
+	isAllChat:function(){
+        // will return is have to load all chat 
+        console.log(Template.instance().isAllChat.get());
```
Comment (renatobecker) [inline app/livechat/client/views/app/tabbar/visitorHistory.js:24]: Please, remove it.

File: app/livechat/client/views/app/tabbar/visitorHistory.js [hunk 8]
```diff
@@ -4,38 +4,84 @@ import moment from 'moment';
...
+		return Template.instance().isAllChat.get();
+	},
+	isChatClicked:function(){
+		// will return that if you have clicked in a single chatHistory
+		return Template.instance().isChatClicked.get();
+	},
+	isfound:function() {
+		// will return if find any search result
+		var isfound= Session.get('found');
+		console.log(isfound)
```
Comment (renatobecker) [inline app/livechat/client/views/app/tabbar/visitorHistory.js:34]: Please, remove it.

File: app/livechat/client/views/app/tabbar/visitorHistory.js [hunk 9]
```diff
@@ -4,38 +4,84 @@ import moment from 'moment';
...
+		return isfound;
+	},
 	isLoading() {
 		return Template.instance().isLoading.get();
 	},
-
+	searchResults(){
+		// will return search result
+		console.log(	Template.instance().searchResult.get())
```
Comment (renatobecker) [inline app/livechat/client/views/app/tabbar/visitorHistory.js:42]: Please, remove it.

File: app/livechat/client/views/app/tabbar/visitorHistory.js [hunk 10]
```diff
@@ -4,38 +4,84 @@ import moment from 'moment';
...
+		len = Template.instance().historyResu.get();
+		return len = len.length-3;
+	},
+	historyResult(){
+		// will return all the messages in history room
+		return Template.instance().historyResu.get().reverse();
+	},
+	time(){
+		return 'hyyy';
+		console.log(datetime);
```
Comment (renatobecker) [inline app/livechat/client/views/app/tabbar/visitorHistory.js:60]: Please, remove it.

File: app/livechat/client/views/app/tabbar/visitorHistory.js [hunk 11]
```diff
@@ -47,17 +93,42 @@ Template.visitorHistory.onCreated(function() {
 		if (!this.visitorId.get() || !currentData || !currentData.rid) {
 			return;
 		}
-
+		console.log('currentData',currentData);
```
Comment (renatobecker) [inline app/livechat/client/views/app/tabbar/visitorHistory.js:96]: Please, remove it.

File: app/livechat/client/views/app/tabbar/visitorHistory.js [hunk 12]
```diff
@@ -47,17 +93,42 @@ Template.visitorHistory.onCreated(function() {
...
+				
+					allHistoryChat.push(count2[j]);
+				
+				
+			}
+		},5000)
+			
+		}
+		console.log('allchathistory',allHistoryChat.length+'count',count.length);
+		console.log("search result array is",allHistoryChat);
```
Comment (renatobecker) [inline app/livechat/client/views/app/tabbar/visitorHistory.js:127]: Please, remove it.

File: app/livechat/client/views/app/tabbar/visitorHistory.js [hunk 13]
```diff
@@ -67,4 +138,50 @@ Template.visitorHistory.events({
...
+		}else{
+			template.isAllChat.set(false);
+			Template.instance().searchResult = new ReactiveVar([]);
+			searchResults = [];
+
+			for(var i=0; i<allHistoryChat.length;i++){
+				if(allHistoryChat[i].msg == text){ // check search input matches with any msg
+					Session.set('found',true);
+					searchResults.push(allHistoryChat[i]); 
+					console.log(allHistoryChat[i]);
```
Comment (renatobecker) [inline app/livechat/client/views/app/tabbar/visitorHistory.js:157]: Please, remove it.

File: app/livechat/client/views/app/tabbar/visitorHistory.js [hunk 14]
```diff
@@ -67,4 +138,50 @@ Template.visitorHistory.events({
...
+		let id = event.currentTarget.id
+		console.log(event);
+		let token = event.currentTarget.attributes.aria.value;
+		console.log('aria-lable'+token);
+				//	let historyArray = template.history.curValue;
+	//	console.log(historyArray);
+		  historyResult  =  await APIClient.v1.get(`livechat/messages.history/${ id }?token=${token}`);
+		  template.historyResu.set(historyResult.messages); 
+		// historyResult = JSON.stringify(historyResult.messages);
+		   console.log('historyResult:'+JSON.stringify(historyResult.messages));
```
Comment (renatobecker) [inline app/livechat/client/views/app/tabbar/visitorHistory.js:182]: Please, remove all of this console.log code.

File: app/models/client/models/ChatMessage.js [hunk 15]
```diff
@@ -3,6 +3,8 @@ import { Mongo } from 'meteor/mongo';
 export const ChatMessage = new Mongo.Collection(null);
 
 ChatMessage.setReactions = function(messageId, reactions) {
+	console.log("chatmessgaes",ChatMessage);
+	console.log(this.update({ _id: messageId }, { $set: { reactions } }))
```
Comment (renatobecker) [inline app/models/client/models/ChatMessage.js:7]: Please, remove it.

File: .meteor/packages (hunk 1 above)
Comment (nitinkumartiwari) [reply .meteor/packages:100]: Removed

File: .meteor/versions (hunk 2 above)
Comment (nitinkumartiwari) [reply .meteor/versions:151]: Removed

File: app/ui/client/views/app/room.js (hunk 3 above)
Comment (nitinkumartiwari) [reply app/ui/client/views/app/room.js:328]: Removed Console Messages

File: app/ui/client/views/app/room.js (hunk 4 above)
Comment (nitinkumartiwari) [reply app/ui/client/views/app/room.js:1197]: Removed Console Messages

File: app/utils/client/lib/RestApiClient.js (hunk 5 above)
Comment (nitinkumartiwari) [reply app/utils/client/lib/RestApiClient.js:12]: Removed Console Messages

File: app/livechat/client/views/app/tabbar/visitorHistory.js (hunk 11 above)
Comment (nitinkumartiwari) [reply app/livechat/client/views/app/tabbar/visitorHistory.js:96]: Removed Console Messages

File: app/livechat/client/views/app/tabbar/visitorHistory.js (hunk 12 above)
Comment (nitinkumartiwari) [reply app/livechat/client/views/app/tabbar/visitorHistory.js:127]: Removed Console Messages

File: app/livechat/client/views/app/tabbar/visitorHistory.js (hunk 13 above)
Comment (nitinkumartiwari) [reply app/livechat/client/views/app/tabbar/visitorHistory.js:157]: Removed Console Messages

File: app/livechat/client/views/app/tabbar/visitorHistory.js (hunk 14 above)
Comment (nitinkumartiwari) [reply app/livechat/client/views/app/tabbar/visitorHistory.js:182]: Removed Console Messages

File: app/models/client/models/ChatMessage.js (hunk 15 above)
Comment (nitinkumartiwari) [reply app/models/client/models/ChatMessage.js:7]: Removed Console Messages

**Comment by renatobecker** [comment]
@nitinkumartiwari I just pulled your last commit, but all of the code that I requested to be removed is still there.

Thanks.

**Comment by nitinkumartiwari** [comment]
Please pull new commit @renatobecker

File: .meteor/packages (hunk 1 above)
Comment (renatobecker) [reply .meteor/packages:100]: No, it's still there.

File: .meteor/versions (hunk 2 above)
Comment (renatobecker) [reply .meteor/versions:151]: No, it's still there.

File: app/livechat/client/views/app/tabbar/visitorHistory.js (hunk 7 above)
Comment (renatobecker) [reply app/livechat/client/views/app/tabbar/visitorHistory.js:24]: it's still there.

File: app/theme/client/main.css [hunk 16]
```diff
@@ -58,3 +58,344 @@
...
+    top:20px;
+}
+.closed-div-p{
+    background: #F5455C;
+    border-radius: 2px;
+    width: 100%;
+    text-align: center;
+    line-height: 0.15em;
+    font-size: 10px;
+    margin:10px 0 15px;
```
Comment (sampaiodiego) [inline app/theme/client/main.css:104]: Codacy Issue found: [Expected an empty line before property "margin" (order/properties-order)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5265440)

Comment (sampaiodiego) [inline app/theme/client/main.css:104]: Codacy Issue found: [Expected single space after ":" with a single-line declaration (declaration-colon-space-after)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5265440)

File: app/theme/client/main.css [hunk 17]
```diff
@@ -58,3 +58,344 @@
...
+	position: absolute;
+	top:12.5%;
+	bottom:12.5%;
+	color: #2F343D;
+	padding: 10px;
+	pointer-events: none;
+	
+}
+.searchresults{
+    margin-top:7%;
```
Comment (sampaiodiego) [inline app/theme/client/main.css:83]: Codacy Issue found: [Expected single space after ":" with a single-line declaration (declaration-colon-space-after)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5265440)

File: app/theme/client/main.css [hunk 18]
```diff
@@ -58,3 +58,344 @@
...
+    position: relative;
+    height: 65px;
+    left:24px;
+    background-color: #ffffff;
+}
+.closed-div{
+    position: relative;
+    top:20px;
+}
+.closed-div-p{
```
Comment (sampaiodiego) [inline app/theme/client/main.css:97]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5265440)

File: app/theme/client/main.css [hunk 19]
```diff
@@ -58,3 +58,344 @@
...
+.closed-div-p span{
+    width: 101px;
+    height:12px;
+    right:138px;
+    top:3px;
+    font-family: Inter;
+    font-style: normal;
+    font-weight: 600;
+    text-align: center;
+    color:#f5455c;
```
Comment (sampaiodiego) [inline app/theme/client/main.css:115]: Codacy Issue found: [Expected single space after ":" with a single-line declaration (declaration-colon-space-after)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5265440)

File: app/theme/client/main.css [hunk 20]
```diff
@@ -58,3 +58,344 @@
...
+.closed-div-p{
+    background: #F5455C;
+    border-radius: 2px;
+    width: 100%;
+    text-align: center;
+    line-height: 0.15em;
+    font-size: 10px;
+    margin:10px 0 15px;
+}
+.closed-div-p span{
```
Comment (sampaiodiego) [inline app/theme/client/main.css:106]: Codacy Issue found: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5265440)

File: app/theme/client/main.css [hunk 21]
```diff
@@ -58,3 +58,344 @@
...
+	width: 345px;
+	height: 40px;
+	background: #ffffff;
+	border: 2px solid #CBCED1;
+	box-sizing: border-box;
+	border-radius: 2px;
+}
+.inner-addon .glyphicon{
+	position: absolute;
+	top:12.5%;
```
Comment (sampaiodiego) [inline app/theme/client/main.css:75]: Codacy Issue found: [Expected single space after ":" with a single-line declaration (declaration-colon-space-after)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5265440)

File: app/theme/client/main.css [hunk 22]
```diff
@@ -58,3 +58,344 @@
...
+    font-style: normal;
+    font-weight: 600;
+    text-align: center;
+    color:#f5455c;
+    background: white;
+    padding-left: 8px;
+    padding-right: 8px;
+}
+.closed-h1 h1{
+    color:#6c727a !important;
```
Comment (sampaiodiego) [inline app/theme/client/main.css:121]: Codacy Issue found: [Expected indentation of 1 tab (indentation)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5265440)

File: app/theme/client/main.css [hunk 23]
```diff
@@ -58,3 +58,344 @@
...
+    height: 65px;
+    left:24px;
+    background-color: #ffffff;
+}
+.closed-div{
+    position: relative;
+    top:20px;
+}
+.closed-div-p{
+    background: #F5455C;
```
Comment (sampaiodiego) [inline app/theme/client/main.css:98]: Codacy Issue found: [Expected indentation of 1 tab (indentation)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5265440)

File: app/theme/client/main.css (hunk 16 above)
Comment (sampaiodiego) [inline app/theme/client/main.css:104]: Codacy Issue found: [Expected indentation of 1 tab (indentation)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5265440)

File: app/theme/client/main.css (hunk 18 above)
Comment (sampaiodiego) [inline app/theme/client/main.css:97]: Codacy Issue found: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5265440)

File: app/theme/client/main.css [hunk 24]
```diff
@@ -58,3 +58,344 @@
...
+
+/* Visitor History CSS */
+.inner-addon{
+	position: relative;
+	left: 24px;
+	width: 345px;
+	height: 40px;
+	background: #ffffff;
+	border: 2px solid #CBCED1;
+	box-sizing: border-box;
```
Comment (sampaiodiego) [inline app/theme/client/main.css:70]: Codacy Issue found: [Expected an empty line before property "box-sizing" (order/properties-order)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5265440)

File: app/theme/client/main.css [hunk 25]
```diff
@@ -58,3 +58,344 @@
...
+    text-align: center;
+    line-height: 0.15em;
+    font-size: 10px;
+    margin:10px 0 15px;
+}
+.closed-div-p span{
+    width: 101px;
+    height:12px;
+    right:138px;
+    top:3px;
```
Comment (sampaiodiego) [inline app/theme/client/main.css:110]: Codacy Issue found: [Expected single space after ":" with a single-line declaration (declaration-colon-space-after)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5265440)

File: app/theme/client/main.css [hunk 26]
```diff
@@ -58,3 +58,344 @@
...
+	position: relative;
+	left: 24px;
+	width: 345px;
+	height: 40px;
+	background: #ffffff;
+	border: 2px solid #CBCED1;
+	box-sizing: border-box;
+	border-radius: 2px;
+}
+.inner-addon .glyphicon{
```
Comment (sampaiodiego) [inline app/theme/client/main.css:73]: Codacy Issue found: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5265440)

File: app/theme/client/main.css [hunk 27]
```diff
@@ -58,3 +58,344 @@
...
+	bottom:12.5%;
+	color: #2F343D;
+	padding: 10px;
+	pointer-events: none;
+	
+}
+.searchresults{
+    margin-top:7%;
+}
+.search-li{
```
Comment (sampaiodiego) [inline app/theme/client/main.css:85]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5265440)

File: app/theme/client/main.css [hunk 28]
```diff
@@ -58,3 +58,344 @@
...
+	box-sizing: border-box;
+	border-radius: 2px;
+}
+.inner-addon .glyphicon{
+	position: absolute;
+	top:12.5%;
+	bottom:12.5%;
+	color: #2F343D;
+	padding: 10px;
+	pointer-events: none;
```
Comment (sampaiodiego) [inline app/theme/client/main.css:79]: Codacy Issue found: [Expected an empty line before property "pointer-events" (order/properties-order)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5265440)

File: app/theme/client/main.css [hunk 29]
```diff
@@ -58,3 +58,344 @@
...
+    background: #F5455C;
+    border-radius: 2px;
+    width: 100%;
+    text-align: center;
+    line-height: 0.15em;
+    font-size: 10px;
+    margin:10px 0 15px;
+}
+.closed-div-p span{
+    width: 101px;
```
Comment (sampaiodiego) [inline app/theme/client/main.css:107]: Codacy Issue found: [Expected indentation of 1 tab (indentation)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5265440)

File: app/livestream/client/styles/liveStreamTab.css [hunk 30]
```diff
@@ -1,4 +1,5 @@
 .contextual-bar__content {
+
```
Comment (sampaiodiego) [inline app/livestream/client/styles/liveStreamTab.css:2]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5265440)

File: app/theme/client/main.css [hunk 31]
```diff
@@ -58,3 +58,344 @@
...
+    height:12px;
+    right:138px;
```
Comment (sampaiodiego) [inline app/theme/client/main.css:109]: Codacy Issue found: [Expected an empty line before property "right" (order/properties-order)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5265440)

File: app/theme/client/main.css [hunk 32]
```diff
@@ -58,3 +58,344 @@
...
+}
+.search-li{
+    display: block;
+    padding-bottom: 3%;
+    position: relative;
+    height: 65px;
+    left:24px;
+    background-color: #ffffff;
+}
+.closed-div{
```
Comment (sampaiodiego) [inline app/theme/client/main.css:93]: Codacy Issue found: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5265440)

File: app/theme/client/main.css (hunk 26 above)
Comment (sampaiodiego) [inline app/theme/client/main.css:73]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5265440)

File: app/theme/client/main.css [hunk 33]
```diff
@@ -58,3 +58,344 @@
...
+	
+}
+.searchresults{
+    margin-top:7%;
+}
+.search-li{
+    display: block;
+    padding-bottom: 3%;
+    position: relative;
+    height: 65px;
```
Comment (sampaiodiego) [inline app/theme/client/main.css:89]: Codacy Issue found: [Expected indentation of 1 tab (indentation)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5265440)

File: app/theme/client/main.css [hunk 34]
```diff
@@ -58,3 +58,344 @@
...
+.inner-addon .glyphicon{
+	position: absolute;
+	top:12.5%;
+	bottom:12.5%;
+	color: #2F343D;
+	padding: 10px;
+	pointer-events: none;
+	
+}
+.searchresults{
```
Comment (sampaiodiego) [inline app/theme/client/main.css:82]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5265440)

File: app/theme/client/main.css (hunk 20 above)
Comment (sampaiodiego) [inline app/theme/client/main.css:106]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5265440)

File: app/theme/client/main.css [hunk 35]
```diff
@@ -58,3 +58,344 @@
...
 @import 'imports/general/rtl.css';
+
+/* Visitor History CSS */
+.inner-addon{
+	position: relative;
+	left: 24px;
+	width: 345px;
+	height: 40px;
+	background: #ffffff;
+	border: 2px solid #CBCED1;
```
Comment (sampaiodiego) [inline app/theme/client/main.css:69]: Codacy Issue found: [Expected "#CBCED1" to be "#cbced1" (color-hex-case)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5265440)

File: app/theme/client/main.css (hunk 16 above)
Comment (nitinkumartiwari) [reply app/theme/client/main.css:104]: done please check

Comment (nitinkumartiwari) [reply app/theme/client/main.css:104]: done please check

File: app/theme/client/main.css (hunk 17 above)
Comment (nitinkumartiwari) [reply app/theme/client/main.css:83]: done please check

File: app/theme/client/main.css (hunk 18 above)
Comment (nitinkumartiwari) [reply app/theme/client/main.css:97]: done please check

File: app/theme/client/main.css (hunk 19 above)
Comment (nitinkumartiwari) [reply app/theme/client/main.css:115]: done please check

File: app/theme/client/main.css (hunk 20 above)
Comment (nitinkumartiwari) [reply app/theme/client/main.css:106]: done please check

File: app/theme/client/main.css (hunk 21 above)
Comment (nitinkumartiwari) [reply app/theme/client/main.css:75]: done please check

File: app/theme/client/main.css (hunk 22 above)
Comment (nitinkumartiwari) [reply app/theme/client/main.css:121]: done please check

File: app/theme/client/main.css (hunk 23 above)
Comment (nitinkumartiwari) [reply app/theme/client/main.css:98]: done please check

File: app/theme/client/main.css (hunk 16 above)
Comment (nitinkumartiwari) [reply app/theme/client/main.css:104]: done please check

File: app/theme/client/main.css (hunk 24 above)
Comment (nitinkumartiwari) [reply app/theme/client/main.css:70]: done please check

File: app/theme/client/main.css (hunk 25 above)
Comment (nitinkumartiwari) [reply app/theme/client/main.css:110]: done please check

File: app/theme/client/main.css (hunk 26 above)
Comment (nitinkumartiwari) [reply app/theme/client/main.css:73]: done please check

File: app/theme/client/main.css (hunk 27 above)
Comment (nitinkumartiwari) [reply app/theme/client/main.css:85]: done please check

File: app/theme/client/main.css (hunk 28 above)
Comment (nitinkumartiwari) [reply app/theme/client/main.css:79]: done please check

File: app/theme/client/main.css (hunk 18 above)
Comment (nitinkumartiwari) [reply app/theme/client/main.css:97]: done please check

File: app/theme/client/main.css (hunk 29 above)
Comment (nitinkumartiwari) [reply app/theme/client/main.css:107]: done please check

File: app/livestream/client/styles/liveStreamTab.css (hunk 30 above)
Comment (nitinkumartiwari) [reply app/livestream/client/styles/liveStreamTab.css:2]: done please check

File: app/theme/client/main.css (hunk 31 above)
Comment (nitinkumartiwari) [reply app/theme/client/main.css:109]: done please check

File: app/theme/client/main.css (hunk 32 above)
Comment (nitinkumartiwari) [reply app/theme/client/main.css:93]: done please check

File: app/theme/client/main.css (hunk 26 above)
Comment (nitinkumartiwari) [reply app/theme/client/main.css:73]: done please check

File: app/theme/client/main.css (hunk 33 above)
Comment (nitinkumartiwari) [reply app/theme/client/main.css:89]: done please check

File: app/theme/client/main.css (hunk 34 above)
Comment (nitinkumartiwari) [reply app/theme/client/main.css:82]: done please check

File: app/theme/client/main.css (hunk 20 above)
Comment (nitinkumartiwari) [reply app/theme/client/main.css:106]: done please check

File: app/theme/client/main.css (hunk 35 above)
Comment (nitinkumartiwari) [reply app/theme/client/main.css:69]: done please check

**Comment by renatobecker** [comment]
@nitinkumartiwari can you please fix the conflict here?
Thanks.

**Comment by luixxiul** [comment]
abandoned?

**Comment by renatobecker** [comment]
Closed in favor of this: https://github.com/RocketChat/Rocket.Chat/pull/17346

**Pull request closed without merging**